db/migrate/20200716170339_add_recipient_id_to_gemini_connection.rb

```diff
@@ -0,0 +1,8 @@
+class AddRecipientIdToGeminiConnection < ActiveRecord::Migration[6.0]
+  def change
+    add_column :gemini_connections, :recipient_id, :string, index: true, unique: true
```

Comment:
Did you want to set a default and allow or disallow null?

Reply:
I don't think so, if there's not a recipient_id created then this field should be null.